src/allotropy/parsers/appbio_quantstudio/appbio_quantstudio_calculated_documents.py

```diff
@@ -30,6 +30,8 @@ def build_quantity(well_item: WellItem) -> CalculatedDocument | None:
         value=quantity,
         data_sources=[
             DataSource(feature="cycle threshold result", reference=well_item),
+            DataSource(feature="Y-intercept", reference=well_item),
+            DataSource(feature="Slope", reference=well_item),
```

Comment:
question: seems like the feature names aren't capitalized elsewhere, should it be `y-intercept` and `slope` throughout this PR, to match?

Reply:
yes, you are right. I change them